robot_radius: 0.22
obstacle_range: 2.5
raytrace_range: 3.0